### PR TITLE
R and Pandas instructions on Showcase Page

### PR DESCRIPTION
**Pull request by anuveyatsu**
- added nav-pills to switch between: 1) just getting raw data; 2) using package with R; 3) using package with Pandas.
- includes some styles
- includes small JS script for 1) preventing default anchor behaviour 2) being able to pass URL with hash so others can jump directly into right place in the page.

### CODE REVIEW

File: app/templates/_snippets.html [hunk 1]
```diff
@@ -80,33 +80,114 @@
...
+$ pip install datapackage
+$ pip install jsontableschema-pandas
+              </code>
+            </pre>
+            <p>To get Data Package run following code:</p>
+            <pre>
+              <code>
+import datapackage
+
+data_url = "https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest"
```
Comment (zelima) [inline app/templates/_snippets.html]: Is this URL correct, should not it be for datapackage.json? Eg: 
`https://bits.datapackaged.com/metadata/{{ dataset.owner }}/{{ dataset.name }}/_v/latest/datapackage.json`

File: app/templates/_snippets.html [hunk 2]
```diff
@@ -80,33 +80,114 @@
...
+
+        <div id="pandas" class="tab-pane fade">
+          <h2>Using in Pandas</h2>
+          <div class="part">
+            <p>In order to work with Data Packages in Pandas you need to install our packages:</p>
+            <pre>
+              <code>
+$ pip install datapackage
+$ pip install jsontableschema-pandas
+              </code>
```
Comment (zelima) [inline app/templates/_snippets.html]: Why are we using `<code>` tag and not proper markdown here? 

We are using `Flask_markdown` right? So we could do something like: 
```
{% filter markdown %}
$ pip install datapackage
$ pip install jsontableschema-pandas
{% endfilter %}
```

See docs for flask-md https://pythonhosted.org/Flask-Markdown/

File: app/templates/_snippets.html [hunk 3]
```diff
@@ -80,33 +80,114 @@
...
+            </pre>
+            <p>To get Data Package run following code:</p>
+            <pre>
+              <code>
```
Comment (zelima) [inline app/templates/_snippets.html]: Same here, let's use markdown?

File: app/templates/_snippets.html (hunk 2 above)
Comment (anuveyatsu) [inline app/templates/_snippets.html]: @zelima we're planning to remove flask-markdown sometimes in the future. We could do it somewhere in the logic using our markdown processor, but there is no need to do so as these instructions are fixed so we can have them in templates.

File: app/templates/_snippets.html (hunk 1 above)
Comment (anuveyatsu) [inline app/templates/_snippets.html]: @zelima oh, I will fix it

**Pull request merged**